src/views/MyProfile/index.js

```diff
@@ -497,7 +497,7 @@ class MyProfile extends Component {
                     justify="flex-start"
                     spacing={2}
                   >
-                    <Grid item xs={12} md={profileCardSize} lg={profileCardSize}>
+                    <Grid item xs={12} md={profileCardSize} lg={8}>
```

Comment:
Is there a symbol that makes sense here instead of 8?  I'm sorry to see a variable replaced with a "magic number".